### PR TITLE
fix: add alignment in tables

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
Align table cells to make cells more readable

## Test Plan
- Badges or icons shoul align center
- Numeric should align right
- String should be align left

## Checklist
- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.
- [ ] If modifying `/webui/react/src/shared/` verify `make -C webui/react test-shared` passes.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/pages/ProjectDetails.tsx [hunk 1]
```diff
@@ -108,7 +108,7 @@ const ProjectDetails: React.FC = () => {
   const [ total, setTotal ] = useState(0);
   const [ canceler ] = useState(new AbortController());
   const pageRef = useRef<HTMLElement>(null);
-  const { canDeleteExperiment, canMoveExperiment, canViewWorkspaces } = usePermissions();
+  const { canDeleteExperiment, canMoveExperiment } = usePermissions();
```
Comment (EmilyBonar) [inline webui/react/src/pages/ProjectDetails.tsx:111]: question: Why was this removed? Seems unrelated to the rest of the PR

**Review by EmilyBonar — APPROVED** [review]
LGTM, just the one question about permissions

File: webui/react/src/pages/ProjectDetails.tsx (hunk 1 above)
Comment (keita-determined) [reply webui/react/src/pages/ProjectDetails.tsx:111]: oh I accidentally removed it. or its not used?
